.travis.yml

```diff
@@ -2,26 +2,36 @@ sudo: required
 
 language: php
 
+php:
+  - '7.0'
+
 services:
   - docker
 
 env:
-  DOCKER_COMPOSE_VERSION: 1.5.2
+  DOCKER_VERSION: 1.10.3-0~trusty
```

Comment:
We should always remember to use fixed version in every repo that uses Docker, and each time when we'd update docker version we'd need to update those hardcoded version as well.